### ISSUE TITLE
How `amp.scale_loss` apply to mutil-model/optimizer

### ISSUE THREAD

**Issue by Jie-Qiao** [NVIDIA/apex#616]
If the loss is generated from different models with different optimizers, how can I apply the code look like below:
```
with amp.scale_loss(loss, optimizer_A,optimizer_B,optimizer_C) as scaled_loss:
    scaled_loss.backward()
```

**Comment by mcarilli** [comment]
This should be supported, see the documentation [here](https://nvidia.github.io/apex/advanced.html#multiple-models-optimizers-losses).  Please reopen if that doesn't work.
